include/xtensor/xchunk_store_manager.hpp

```diff
@@ -84,23 +106,59 @@ namespace xt
                     path.append(".");
                 path.append(std::to_string(*it));
             }
```

Comment:
I am not sure storing things in a string and comparing strings in std::find is the way to go.

Reply:
maybe keeping indices in an `std::vector`.